lib/keyring.js

```diff
@@ -34,16 +34,24 @@ KeyRing.DEFAULTS = {
 };
 
 /**
-  * Migrate that old shit to new shit
+  * Migrate the old keyring to new key.ring folder
+  * @param {String} oldPath - path to the old keyring
```

Comment:
Add a `@private` line to the comment block so this method doesn't show up in the docs.